docs/pages/modules/autolinking.mdx

```diff
@@ -143,6 +143,20 @@ The following config in **package.json** will exclude `expo-random` from autolin
 }
 ```
 
+Note that the above `excludes` option is specifically for excluding expo modules. For excluding the autolinking for any other packages, use the `react-native.config.js` at the root directory of your project:
```

Comment:
```suggestion
Note that the `exclude` option is for excluding Expo packages. To exclude the autolinking for any other packages, create **react-native.config.js** at the root directory of your project and apply the configuration as shown in the example below:
```

- Let's be clear about creating **react-native.config.js** file and describe that it's an example
- Suggestion to apply other formatting for consistency and fixing the typo